### ISSUE TITLE
Cannot install it for Xamarin Portable Library

### ISSUE THREAD

**Issue by tylerlong** [tmenier/Flurl#118]
How to reproduce:

Create a portable library project with Xamarin Studio, then try to install this library.

![screen shot 2016-07-20 at 10 02 49 pm](https://cloud.githubusercontent.com/assets/733544/16989188/d2083f6c-4ec5-11e6-9131-e1fa526203e0.png)

```
Adding Flurl...
Adding 'Flurl 2.1.0' to XXX.
Could not install package 'Flurl 2.1.0'. You are trying to install this package into a project that targets 'portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarintvos10+xamarinwatchos10+xamarinios10', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.
```

**Comment by kroniak** [comment]
@tylerlong Did you install beta version in thread #79?

**Comment by kroniak** [comment]
@tylerlong I don't have Xamarin Studio and I have no problem with Android with VS2015  #119. Do you know what PCL version make Xamarin Studio?

**Comment by tylerlong** [comment]
@kroniak It is the default protable project created by Xamarin Studio without changing the targets.

The targets are: portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarintvos10+xamarinwatchos10+xamarinios10

**Comment by tmenier** [comment]
Does #119 resolve this or is it a separate issue? I'm otherwise ready to go with Flurl 2.2 but if this one's an easy fix I can wait on it.

**Comment by tmenier** [comment]
https://github.com/tmenier/Flurl/releases/tag/Flurl.2.2.0-beta1

I think the Url.Combine change was significant enough for a minor version bump.

**Comment by tylerlong** [comment]
Could you please release a beta version to NuGet? I can have a quick try then. For now I have no idea how can I install it.

**Comment by kroniak** [comment]
@tmenier Please delete tag, I am working at this issue yet.
I will say when all will be ready and tested.

@tylerlong After tests.

**Comment by kroniak** [comment]
@tylerlong Please test new beta version on https://www.myget.org/F/flurl/api/v3/index.json feed.

**Comment by tmenier** [comment]
https://www.nuget.org/packages/Flurl/2.2.0-beta1

**Comment by kroniak** [comment]
Closed
